modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -82,10 +82,14 @@ provisioning scripts to accommodate your organization's requirements.
 *** The subnet range used for Runtime Fabric in CIDR notation
 *** If virtual machines are allowed, public IP addresses
 *** How to gain shell access to each VM
-** Determine if inbound traffic should be enabled for Anypoint Runtime Fabric. This is based on the function of your organization's Mule applications. To enable inbound traffic, you must:
-*** Have a TLS certificate with the desired Common Name set to route traffic to Runtime Fabric.
-*** Be able to create or configure an external TCP load balancer to manage traffic to applications running in Runtime Fabric.
-*** Be able to configure DNS settings to associate the Common Name on the TLS certificate to the IP address of the external load balancer.
+
+== Inbound Traffic Enablement Requirements
+
+Determine if inbound traffic should be enabled for Anypoint Runtime Fabric. This is based on the function of your organization's Mule applications. 
+
+*** If you plan to configure a TLS context using PEM or JKS certificates, you must have the necessary permissions for the Runtime Fabric environment you want to use for applications: Refer to xref:anypoint-security::configure-tls-context-rtf-lb.adoc[Configure a TLS Context for Anypoint Runtime Fabric] for details about the permissions needed.
```

Comment:
With the Multiple Virtual Servers UI, no additional permissions are needed beyond Manage Runtime Fabrics (I think, I will check to make sure). Either way, that doc is specific to Secrets Manager which is no longer required for enabling inbound traffic.

Reply:
Fixed this.